content/docs/rfcs/14/README.md

```diff
@@ -132,6 +132,28 @@ If an optional attribute, such as `meta`, is absent, the concatenation of attrib
 
 This hashing schema is deemed appropriate for use cases where a cross-implementation deterministic hash is needed, such as message deduplication and integrity validation. The collision probability offered by this hashing schema can be considered negligible. This is due to the deterministic concatenation order of the message attributes, coupled with using a SHA-2 (256-bit) hashing algorithm.
 
+### Test vectors
+
+Waku message hash computation:
+```
+pubsub_topic = 0x2f77616b752f322f64656661756c742d77616b752f70726f746f
+message.payload = 0x010203045445535405060708
```

Comment:
Might be worth including an "empty payload" example. While payload is not optional, it can be an empty byte array. Seems like an edge case that would be appropriate to include just in case.